docs/sources/operations/scalability.md

```diff
@@ -27,3 +27,63 @@ The query scheduler process itself can be started via the `-target=query-schedul
 In compute-constrained environments, garbage collection can become a significant performance factor. Frequently-run garbage collection interferes with running the application by using CPU resources. The use of memory ballast can mitigate the issue. Memory ballast allocates extra, but unused virtual memory in order to inflate the quantity of live heap space. Garbage collection is triggered by the growth of heap space usage. The inflated quantity of heap space reduces the perceived growth, so garbage collection occurs less frequently.
 
 Configure memory ballast using the ballast_bytes configuration option.
+
+## Remote Rule Evaluation
+
+This feature was introduced as part of [`LID-0002`](https://github.com/grafana/loki/pull/8129).
+
+### Background & Identification of Problem
```

Comment:
Your overview text is fine, but I would remove both of the headings:
"Background & Identification of Problem" 
"Recommended Solution"
You're documenting a feature here, not a bug fix.  All of the paragraph text from lines 37 - 51 can sit right under the "Remote rule evaluation" heading.

Reply:
OK 👍